### ISSUE TITLE
Release of v0.28.0

### ISSUE THREAD

**Issue by Lazar955** [lncm/docker-bitcoind#104]
Is there a plan to release a Docker image for v0.28.0?

**Comment by nolim1t** [comment]
Yes of course eventually. The community is welcome to create pull requests which we've been now relying on lately
